src/services/ebsSnapshot/format.ts

```diff
@@ -0,0 +1,81 @@
+import { generateUniqueId } from '@cloudgraph/sdk'
+
+import { AwsEbsSnapshot } from '../../types/generated'
+import t from '../../properties/translations'
+import { formatTagsFromMap } from '../../utils/format'
+import { ebsSnapshotArn } from '../../utils/generateArns'
+import { RawAwsEBSSnapshot } from './data'
+
+/**
+ * EBS Snapshot
+ */
+
+export default ({
+  service: rawData,
+  account,
+  region,
+}: {
+  service: RawAwsEBSSnapshot
+  account: string
+  region: string
+}): AwsEbsSnapshot => {
+  const {
+    DataEncryptionKeyId: dataEncryptionKeyId,
+    Description: description,
+    Encrypted: encrypted,
+    KmsKeyId: kmsKeyId,
```

Comment:
Can we make a connection for kms key here?

Reply:
@tyler-dunkel KMS connection added